src/Utopia/Messaging/Adapters/Email/Email.php

```diff
@@ -0,0 +1,43 @@
+<?php
```

Comment:
It will make more sense to me to have this class located in: `src/Utopia/Messaging/Adapters/Email.php` while the Email providers themselves can stay at the same location.